usermods/seven_segment_display_reloaded_v2/readme.md

```
@@ -0,0 +1,281 @@
+  - LED-Numbers-Seconds
+  - LED-Numbers-Colons
+  - LED-Numbers-Light
+  - LED-Numbers-Day
+  - LED-Numbers-Month
+  - LED-Numbers-Year
+
+## Example LED Definitions
+
+The following is an example of an LED layout for a seven segment display. The diagram below shows the segment positions:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)